include/imu-core/imu_interface.hpp

```diff
@@ -7,6 +7,8 @@
 #include <Eigen/Eigen>
 #include "real_time_tools/usb_stream.hpp"
 
+
+
```

Comment:
If possible, try not to commit whitespace/newline changes until they are necessary / you are doing a code formation.